.github/workflows/release-post-merge.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           fi
 
           echo "VERSION:$version"
-          tag="v${version}"
+          tag="${version}"
```

Comment:
This will change our git tag and release name. Is it ok? https://github.com/mongodb/mongodb-atlas-kubernetes/actions/runs/3969499878/workflow#L83

Reply:
Fixed